pkg/collector/corechecks/servicediscovery/module/config.go

```diff
@@ -19,6 +19,7 @@ const discoveryNS = "discovery"
 
 type discoveryConfig struct {
 	cpuUsageUpdateDelay time.Duration
+	ignoreComms         string
```

Comment:
why is it represented as string and not `[]string`? or `map[string]struct{}`? why do we need to defer the processing of it?

Reply:
I've changed it, now this struct keeps a map of ignored commands.